hybridCarboniteMetadataService/src/main/scala/cromwell/services/metadata/hybridcarbonite/HybridCarboniteConfig.scala

```diff
@@ -0,0 +1,34 @@
+package cromwell.services.metadata.hybridcarbonite
+
+import akka.actor.ActorSystem
+import com.typesafe.config.Config
+import common.validation.Validation._
+import cromwell.core.{WorkflowId, WorkflowOptions}
+import cromwell.core.filesystem.CromwellFileSystems
+import cromwell.core.path.{PathBuilderFactory, PathFactory}
+import cromwell.core.path.PathFactory.PathBuilders
+
+import scala.concurrent.Await
+import scala.concurrent.duration._
+
+final case class HybridCarboniteConfig(pathBuilders: PathBuilders, bucket: String) {
+  def makePath(workflowId: WorkflowId)= PathFactory.buildPath(HybridCarboniteConfig.pathForWorkflow(workflowId, bucket), pathBuilders)
+}
+
+object HybridCarboniteConfig {
+
+  def pathForWorkflow(id: WorkflowId, bucket: String) = s"gs://$bucket/$id/$id.json"
```

Comment:
What the point of the `/$id/` if the json file also has the id in its name?

Reply:
See the "The object name in GCS shall be..." section in https://docs.google.com/document/d/1VYnzk97yTtllozO9ivZpZQTwrsY5T0wGqxlvAbrEQgg/edit#